src/components/flow/actions/openticket/OpenTicket.tsx

```diff
@@ -1,17 +1,21 @@
 import * as React from 'react';
 import { OpenTicket } from 'flowTypes';
-import { renderAsset } from '../helpers';
-import { AssetType } from 'store/flowContext';
 import { fakePropType } from 'config/ConfigProvider';
 
-const OpenTicketComp: React.SFC<OpenTicket> = ({ ticketer }, context: any): JSX.Element => {
-  return renderAsset(
-    {
-      id: ticketer.uuid,
-      name: ticketer.name,
-      type: AssetType.Ticketer
-    },
-    context.config.endpoints
+const OpenTicketComp: React.SFC<OpenTicket> = (
+  { ticketer, subject },
+  context: any
+): JSX.Element => {
+  const showTicketer = ticketer.name.indexOf(context.config.brand) === -1;
```

Comment:
maybe more reliable to check the ticketer type? Just thinking that Zendesk and Mailgun ticketer names could actually include strings like rapidpro and textit.. if that's what people call their accounts on those services.

Reply:
Yes. Meant to add a comment here, this is super goofy.

The definition doesn't include the ticketer type. Looked at adding it to the ui metadata, but the editor deals with generic asset selection with name/uuid here and would be a big refactor to change that.

Reply:
Not to mention the rendering of tickets only has the context of the current action. This switch is simply for whether we show the subtext of which ticketer it is opening a ticket against. So I think it is okay if this is hidden in that edge case for now.

Reply:
Alternatively, I considered just always showing the ticketer used, but that feels worse for the common case.

Reply:
I thought the plan was to hide/show it based on just whether there are > 1 ticketers ?

Reply:
You are confusing this with the form. This is the rendering of the flow. The form select shows if there are > 1 ticketer or if the currently selected ticketer no longer exists.

Reply:
aaaahhhh in which case this is goofy but completely fine